src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/Binders/ByteArrayModelBinder.cs

```diff
@@ -5,10 +5,10 @@
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc.Internal;
 
-namespace Microsoft.AspNetCore.Mvc.ModelBinding
+namespace Microsoft.AspNetCore.Mvc.ModelBinding.Binders
 {
     /// <summary>
-    /// ModelBinder to bind Byte Arrays.
+    /// ModelBinder to bind byte Arrays.
     /// </summary>
     public class ByteArrayModelBinder : IModelBinder
```

Comment:
Yeah, you're right. Every `byte[]` is handled here so e.g. `?byte[0]=1&byte[1]=2&...` wouldn't have worked before. Does make me wonder what the MVC 5 / Web API behaviours were...